app/src/main/java/org/jellyfin/androidtv/ui/presentation/CardPresenter.java

```diff
@@ -410,7 +427,25 @@ public void onBindViewHolder(Presenter.ViewHolder viewHolder, Object item) {
             }
         }
 
-        holder.updateCardViewImage(rowItem.getImageUrl(holder.mCardView.getContext(), mImageType, holder.getCardHeight()));
+        BitmapDrawable blurHashDrawable = null;
+        if (rowItem.getBaseItem() != null && rowItem.getBaseItem().getImageBlurHashes() != null) {
+            HashMap<String, String> blurHashMap;
+            if (aspect == ASPECT_RATIO_BANNER) {
+                blurHashMap = rowItem.getBaseItem().getImageBlurHashes().get(org.jellyfin.apiclient.model.entities.ImageType.Banner);
+            } else if (aspect == ImageUtils.ASPECT_RATIO_16_9 && !isUserView) {
+                blurHashMap = rowItem.getBaseItem().getImageBlurHashes().get(org.jellyfin.apiclient.model.entities.ImageType.Thumb);
+            } else {
+                blurHashMap = rowItem.getBaseItem().getImageBlurHashes().get(org.jellyfin.apiclient.model.entities.ImageType.Primary);
+            }
+
+            if (blurHashMap != null) {
+                String blurHash = (String) blurHashMap.values().toArray()[0];
```

Comment:
I'm not sure if just taking the first value is the way to go. AFAIK you need to get the blurhash for the current image id. You also want to verify if the hashmap is not empty because the server sometimes doesn't include blurhashes.

Reply:
You're right, episode thumbs and season posters have arrays with two values—one is the blurhash for the image and the other is the blurhash for the parent image. I didn't notice because a lot of times the parent blurhash is second, but sometimes they're flipped, so this doesn't work. Using image tags works.